docs/readthedocs/requirements-doc.txt

```diff
@@ -10,7 +10,7 @@ cloudpickle==2.1.0
 ray[tune]==1.9.2
 ray==1.9.2
 torch==1.9.0
-Pygments==2.3.1
+Pygments==2.6.1
```

Comment:
This change is because of the bug met here during local build of the sphinx project:
https://stackoverflow.com/questions/63840334/fix-some-very-minor-issues-and-rebuild-sphinx-but-got-the-error-wrong-color-for